web/client/codechecker_client/cmd_line_client.py

```diff
@@ -1139,6 +1138,48 @@ def print_gerrit_json(reports, output_dir):
         LOG.info("Gerrit review file was created: %s\n",
                  gerrit_review_json)
 
+    def print_codeclimate_json(reports, output_dir):
+        """ Converts the given reports to codeclimate format.
+
+        This function will convert the given report to Code Climate format
+        and writes the issues to the given output directory to a JSON file.
```

Comment:
@gyorb In case of json we just print the json to the standard output. In case of gerrit and codeclimate we write the results to a file.

I think the reason why we write the results to a file was that multiple output type can be given to the diff command, for example `html`, `gerrit`, `json`. In this case if we print the gerrit and json output to the standard output it will be an invalid json.

I agree that it is not consistent because why do we print the json to the standard output and why don't we print the gerrit results to the standard output too?
What do you think, what would be the best solution for this?

Reply:
I see the problem with the multiple output types. Keep it as it is for now, but please extend the help message and the documentation so the user can see what output to expect in what case.
```
-o codeclimate  a codeclimate.json file will be generated
...
```
In this case the export directory is required right?

Reply:
Okay, I have extended the documentation. Yes, the export directory is required in case of gerrit, html and codeclimate outputs.